.vuepress/config.js

```diff
@@ -168,6 +164,22 @@ module.exports = {
             '/learn/security/tenant_tokens',
           ],
         },
+        {
+          title: '🔢 Updates and versioning',
+          path: '/learn/advanced/asynchronous_operations.html',
+          collapsable: false,
+          children: [
+            '/learn/advanced/previous_docs_version',
```

Comment:
Since it's "Update and migration", should we start with the updating guide, the migration guide, and then the versioning guide? Or maybe alphabetical order is better 🤔